aviary/variable_info/variable_meta_data.py

```diff
@@ -1196,47 +1197,22 @@
     desc='part 25 structural category',
 )
 
-add_meta_data(
-    Aircraft.Design.RESERVES,
-    meta_data=_MetaData,
-    historical_name={"GASP": 'INGASP.FRESF',
-                     "FLOPS": None,
-                     "LEAPS1": None
-                     },
-    option=True,
-    units="lbm",
-    desc='required fuel reserves: directly in lbm',
-    default_value=0,
-)
-
 add_meta_data(
     Aircraft.Design.RESERVES_FRACTION,
     meta_data=_MetaData,
-    historical_name={"GASP": 'INGASP.FRESF2',  # the real name is INGASP.FRESF
+    historical_name={"GASP": None,
                      "FLOPS": None,
                      "LEAPS1": None
                      },
     option=True,
     units="unitless",
-    desc='required fuel reserves: given as a proportion of mission fuel',
+    desc='required fuel reserves: given as a proportion of mission fuel. This value must be nonnegative.\
+          If it is 0.5, the reserve fuel is half of the mission fuel (one third of the total fuel). Note\
+          it can be greater than 1. If it is 2, there would be twice as much reserve fuel as mission fuel\
+          (the total fuel carried would be 1/3 for the mission and 2/3 for the reserve)',
     default_value=0,
 )
 
-add_meta_data(
-    Aircraft.Design.RESERVES_OPTION,
-    meta_data=_MetaData,
-    historical_name={"GASP": 'INGASP.FRESF3',  # the real name is INGASP.FRESF
-                     "FLOPS": None,
-                     "LEAPS1": None
-                     },
-    option=True,
-    types=Reserves_Type,
-    units="unitless",
-    desc=f'if {Reserves_Type.Set_Direct}, use Aircraft.Design.RESERVES; \
-           if {Reserves_Type.Set_Fraction}, use Aircraft.Design.RESERVES_FRACTION',
-    default_value=Reserves_Type.Set_None,
-)
```

Comment:
Leave this in, but change the historical_name for GASP to `None`, the description to `"A flag that is set internally to switch between using a fixed reserve fuel mass or a fraction of mission fuel."` and default_value to `Set_Direct`

Reply:
Jason and Ken want to remove it. @jkirk5 and @Kenneth-T-Moore , what do you think? Should I keep it?